apps/app/components/InfiniteMailbox/index.tsx

```diff
@@ -109,12 +113,15 @@ const InfiniteBox: ForwardRefRenderFunction<
 
   const loaderEl = useMemo(() => loader || <Box>Loading</Box>, [loader])
 
-  const dataMsg: MessageItem[] = useMemo(() => {
+  const dataOriginFlat = useMemo(() => {
     if (!data) return []
-    const dataList = data.pages.map((item) => item.messages).flat()
+    return data.pages.map((item) => item.messages).flat()
```

Comment:
```suggestion
    return data?.pages.map((item) => item.messages).flat() || []
```
In this way, `if` can be omitted